pxtsim/runtime.ts

```diff
@@ -195,8 +197,8 @@ namespace pxsim {
 
         private poke() {
             this.lock = true;
-            let top = this.events.shift()
-            this.runtime.runFiberAsync(this.handler, top)
+            const value = this.events.shift();
+            this.runtime.runFiberAsync(this.handler, ...(this.valueToArgs ? this.valueToArgs(value) : [value]))
```

Comment:
where is the "this.handler" field defined?

Reply:
It's the RefAction associated with the event via EventBus.listen(). That didn't change, well I did rename the variable _handler instead of mHandler but the property is the same.

Reply:
Mis-read the question. It's defined below in the EventQueue class.

Reply:
Class member-ordering is not defined in the lint rules :(

Reply:
We are passing ``[value]`` instead of ``value`` to the fiber. Is that expected?